src/main/webapp/resources/js/components/samples/components/SampleFiles.tsx

```diff
@@ -87,14 +78,69 @@ export function SampleFiles() {
     }
   }, [filesToUpload]);
 
+  /*
+  Function to get sample files from the server and dispatch to the store
+ */
+  const getSampleFiles = () => {
+    fetchSampleFiles({
+      sampleId: sample.identifier,
+      projectId,
+    })
+      .then((data) => {
+        dispatch(setSampleFiles(data));
+      })
+      .catch((e) => notification.error({ message: e }));
+  };
+
   /*
   Custom function to upload sequence, assembly, and fast5 files uploaded
   using the ant design upload (DragUpload) component
    */
   const uploadFiles = () => {
+    const controller = new AbortController();
```

Comment:
Why not just initialize the abortController as a `const` variable above instead of a state.  You are never clearing it.  Also wonder if we could make this a reusable hook somehow?

Reply:
Updated in 85b90900deaddee54dbae6a8198ae5b1b77a577c